app/1.3.x/db-less-admin-api.md

```diff
@@ -251,41 +251,69 @@ certificate_data: |
+
+ca_certificate_json: |
+    {
+        "id": "7fca84d6-7d37-4a74-a7b0-93e576089a41",
+        "created_at": 1422386534,
+        "cert": "-----BEGIN CERTIFICATE-----...",
+        "tags": ["user-level", "low-priority"]
+    }
+
+ca_certificate_data: |
+    "data": [{
+        "id": "d044b7d4-3dc2-4bbc-8e9f-6b7a69416df6",
+        "created_at": 1422386534,
+        "cert": "-----BEGIN CERTIFICATE-----...",
+        "tags": ["user-level", "low-priority"]
+    }, {
+        "id": "a9b2107f-a214-47b3-add4-46b942187924",
+        "created_at": 1422386534,
+        "cert": "-----BEGIN CERTIFICATE-----...",
+        "tags": ["admin", "high-priority", "critical"]
+    }],
+
 sni_body: |
     Attributes | Description
     ---:| ---
     `name` | The SNI name to associate with the given certificate.
     `tags`<br>*optional* |  An optional set of strings associated with the SNIs, for grouping and filtering. 
-    `certificate` |  The id (a UUID) of the certificate with which to associate the SNI hostname. The Certificate must have a valid private key associated with it to be used by the SNI object.  With form-encoded, the notation is `certificate.id=<certificate_id>`. With JSON, use `"certificate":{"id":"<certificate_id>"}`.
+    `certificate` |  The id (a UUID) of the certificate with which to associate the SNI hostname  With form-encoded, the notation is `certificate.id=<certificate_id>`. With JSON, use `"certificate":{"id":"<certificate_id>"}`.
 
 sni_json: |
     {
-        "id": "7fca84d6-7d37-4a74-a7b0-93e576089a41",
+        "id": "04fbeacf-a9f1-4a5d-ae4a-b0407445db3f",
         "name": "my-sni",
         "created_at": 1422386534,
         "tags": ["user-level", "low-priority"],
-        "certificate": {"id":"d044b7d4-3dc2-4bbc-8e9f-6b7a69416df6"}
+        "certificate": {"id":"43429efd-b3a5-4048-94cb-5cc4029909bb"}
     }
 
 sni_data: |
     "data": [{
-        "id": "a9b2107f-a214-47b3-add4-46b942187924",
+        "id": "d26761d5-83a4-4f24-ac6c-cff276f2b79c",
         "name": "my-sni",
         "created_at": 1422386534,
         "tags": ["user-level", "low-priority"],
-        "certificate": {"id":"04fbeacf-a9f1-4a5d-ae4a-b0407445db3f"}
+        "certificate": {"id":"91020192-062d-416f-a275-9addeeaffaf2"}
     }, {
-        "id": "43429efd-b3a5-4048-94cb-5cc4029909bb",
+        "id": "a2e013e8-7623-4494-a347-6d29108ff68b",
         "name": "my-sni",
         "created_at": 1422386534,
         "tags": ["admin", "high-priority", "critical"],
-        "certificate": {"id":"d26761d5-83a4-4f24-ac6c-cff276f2b79c"}
+        "certificate": {"id":"147f5ef0-1ed6-4711-b77f-489262f8bff7"}
     }],
 
 upstream_body: |
     Attributes | Description
     ---:| ---
     `name` | This is a hostname, which must be equal to the `host` of a Service.
+    `algorithm`<br>*optional* | What kinds of load balancing algorithm to use. One of: `round-robin`, `consistent-hashing`, or `least-connections`. Defaults to `"consistent"`.
```

Comment:
This <tt>Defaults to \`"consistent"\`</tt> is incorrect (default is `round-robin`) and seems to indicate you ran the generator on an outdated pull of `next` (before the latest changes to the `algorithm` field were committed.

Reply:
Sorry it was indeed outdated. Pushed a new commit.